examples/systems_of_equations/systems_of_equations_ex9/Makefile.am

```diff
@@ -0,0 +1,11 @@
+example_name  = systems_of_equations_ex9
+check_SCRIPTS = run.sh
+install_dir   = $(examples_install_path)/systems_of_equations/ex9
+data          = systems_of_equations_ex9.C run.sh periodiccomponent.exo
+sources       = $(data) run.sh
+
+CLEANFILES = solution.exo
+
```

Comment:
Have a look at miscellaneous_ex9's Makefile.am. I think you need a section in the Makfile.am like the following to ensure that the input mesh gets linked to the build directory.
```
# also need to link files for VPATH builds
if LIBMESH_VPATH_BUILD
  BUILT_SOURCES = .linkstamp
.linkstamp:
        -rm -f miscellaneous_ex9.exo && $(LN_S) -f $(srcdir)/miscellaneous_ex9.exo
        $(AM_V_GEN)touch .linkstamp

  CLEANFILES += miscellaneous_ex9.exo .linkstamp
endif
```